src/main/resources/init_db.sql

```diff
@@ -1,20 +1,44 @@
-CREATE SCHEMA IF NOT EXISTS `taxi_service` DEFAULT CHARACTER SET utf8;
-USE `taxi_service`;
+CREATE SCHEMA IF NOT EXISTS `taxi` DEFAULT CHARACTER SET utf8;
+USE `taxi`;
 
 DROP TABLE IF EXISTS `manufacturers`;
 CREATE TABLE `manufacturers` (
-                                        `id` BIGINT(11) NOT NULL AUTO_INCREMENT,
-                                        `name` VARCHAR(225) NOT NULL,
-                                        `country` VARCHAR(225) NOT NULL,
-                                        `is_deleted` TINYINT NOT NULL DEFAULT 0,
-                                        PRIMARY KEY (`id`));
+`id` BIGINT(11) NOT NULL AUTO_INCREMENT,
+`name` VARCHAR(225) NOT NULL,
+`country` VARCHAR(225) NOT NULL,
+`is_deleted` TINYINT NOT NULL DEFAULT 0,
+PRIMARY KEY (`id`));
 
 DROP TABLE IF EXISTS `drivers`;
 CREATE TABLE `drivers` (
-                                  `id` BIGINT(11) NOT NULL AUTO_INCREMENT,
-                                  `name` VARCHAR(225) NOT NULL,
-                                  `license_number` VARCHAR(225) NOT NULL,
-                                  `is_deleted` TINYINT NOT NULL DEFAULT 0,
-                                  PRIMARY KEY (`id`),
-                                  UNIQUE INDEX `id_UNIQUE` (id ASC) VISIBLE,
-                                  UNIQUE INDEX `license_number_UNIQUE` (`license_number` ASC) VISIBLE);
+`id` BIGINT(11) NOT NULL AUTO_INCREMENT,
+`name` VARCHAR(225) NOT NULL,
+`license_number` VARCHAR(225) NOT NULL,
+`is_deleted` TINYINT NOT NULL DEFAULT 0,
+ PRIMARY KEY (`id`),
+UNIQUE INDEX `id_UNIQUE` (id ASC) VISIBLE,
+UNIQUE INDEX `license_number_UNIQUE` (`license_number` ASC) VISIBLE);
+
+DROP TABLE IF EXISTS `cars`;
+CREATE TABLE `cars` (
+`id` BIGINT NOT NULL AUTO_INCREMENT,
+`model` VARCHAR(255) NULL,
+`is_deleted` TINYINT NOT NULL DEFAULT 0,
+`manufacturer_id` BIGINT NULL,
+PRIMARY KEY (`id`),
+CONSTRAINT `cars_manufacturers_fk`
+FOREIGN KEY (`manufacturer_id`)
+REFERENCES `taxi_service`.`manufacturers` (`id`)
+ON DELETE NO ACTION
+ON UPDATE NO ACTION);
```

Comment:
formatting for better readability can be like this:
```suggestion
DROP TABLE IF EXISTS `cars`;
CREATE TABLE `cars` (
    `id` BIGINT NOT NULL AUTO_INCREMENT, 
    `model` VARCHAR(255) NULL,
    `is_deleted` TINYINT NOT NULL DEFAULT 0,
    `manufacturer_id` BIGINT NULL,
    PRIMARY KEY (`id`),
    CONSTRAINT `cars_manufacturers_fk`
         FOREIGN KEY (`manufacturer_id`)
              REFERENCES `taxi_service`.`manufacturers` (`id`)
              ON DELETE NO ACTION
              ON UPDATE NO ACTION);
```